### PR TITLE
fix: docs build

### PR DESCRIPTION
**Pull request by zepatrik**
## Related issue

Malformed sidebar

### CODE REVIEW

File: docs/sidebar.json [hunk 1]
```diff
@@ -61,8 +61,6 @@
     "reference/html-forms",
     "reference/api",
     {
-      "type": "category",
-      "label": "Command Line Interface (CLI)",
       "Command Line Interface (CLI)": [
```
Comment (aeneasr) [inline docs/sidebar.json:64]: This won't fix the docs. Revert the changes and:

```suggestion
      "items": [
```

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: docs/sidebar.json (hunk 1 above)
Comment (zepatrik) [reply docs/sidebar.json:64]: Why would it not?

https://docusaurus.io/docs/sidebar#creating-a-hierarchy

> Note: it's possible to use the shorthand syntax to create nested categories:
> ```
> module.exports = {
>   docs: {
>     Guides: [
>       'creating-pages',
>       {
>         Docs: [
>           'docs-introduction',
>           'docs-sidebar',
>           'markdown-features',
>           'versioning',
>         ],
>       },
>     ],
>   },
> };
> ```

Comment (aeneasr) [reply docs/sidebar.json:64]: Looks like I learned something new ;)
